### PR TITLE
Add atreides02a

### PR DESCRIPTION
**Pull request by abcdefg30**

### CODE REVIEW

File: mods/d2k/maps/atreides-02a/atreides02a.lua [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	{ "trike", "trike" },
+	{ "light_inf", "light_inf", "light_inf" },
+	{ "light_inf", "trike" },
+	{ "trike", "trike" }
+}
+
+HarkonnenAttackDelay = { }
+HarkonnenAttackDelay["Easy"] = DateTime.Minutes(5)
+HarkonnenAttackDelay["Normal"] = DateTime.Minutes(2) + DateTime.Seconds(40)
+HarkonnenAttackDelay["Hard"] = DateTime.Minutes(1) + DateTime.Seconds(20)
```
Comment (HenrytheSlav) [inline mods/d2k/maps/atreides-02a/atreides02a.lua]: Is DateTime.Minutes(1) + DateTime.Seconds(20) your prefered over DateTime.Seconds(80) or is it stylistically better? I want to know so we can follow same guidelines across the files.

File: mods/d2k/maps/atreides-02a/map.yaml [hunk 2]
```diff
@@ -0,0 +1,220 @@
+MapFormat: 7
+
+RequiresMod: d2k
+
+Title: Atreides 02a
+
+Description: Infiltrate the Imperial Basin and build up our forces until they are strong enough to eradicate the local Harkonnen presence.\n\nThe Harkonnen are reinforceing their troops by air, so be on your guard. Use the Outpost's radar to detect attacks from unexpected quarters.\n\nBe careful when minig the Spice. Spice mounds grow out of the sand. While a vital source of Spice, Spice mounds can damage or destroy any unit that blunders into them.\n\nGood luck.
```
Comment (HenrytheSlav) [inline mods/d2k/maps/atreides-02a/map.yaml]: reinforceing -> reinforcing
I believe this is the correct form, google agrees too.

File: mods/d2k/maps/atreides-02a/atreides02a.lua (hunk 1 above)
Comment (abcdefg30) [inline mods/d2k/maps/atreides-02a/atreides02a.lua]: I just prefer it over `DateTime.Seconds(80)`.

File: mods/d2k/maps/atreides-02a/map.yaml (hunk 2 above)
Comment (abcdefg30) [inline mods/d2k/maps/atreides-02a/map.yaml]: Ops, thanks.

**Comment by penev92** [comment]
Two things at first glance:
 - [ ] The mission video is missing.
 - [ ] The camera isn't centered on my base at the start of the mission.

**Comment by abcdefg30** [comment]
Updated.

File: mods/d2k/maps/atreides-02a/map.yaml [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+	PlayerReference@Atreides:
+		Name: Atreides
+		Playable: True
+		LockFaction: True
+		Faction: atreides
+		LockColor: True
+		ColorRamp: 170,255,127
```
Comment (Mailaender) [inline mods/d2k/maps/atreides-02a/map.yaml]: `170,255,200` for consistency please.

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Comment by abcdefg30** [comment]
Updated.

**Comment by abcdefg30** [comment]
Updated `map.png` with the changes made in #9478.
(Made it 32bit and pngcrushed it.)

File: mods/d2k/maps/atreides-02a/atreides02a.lua [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+HarkonnenAttackWaves = { }
+HarkonnenAttackWaves["Easy"] = 3
+HarkonnenAttackWaves["Normal"] = 6
+HarkonnenAttackWaves["Hard"] = 9
```
Comment (obrakmann) [inline mods/d2k/maps/atreides-02a/atreides02a.lua]: Just a note for the future: you can write it like this as well, which looks a bit nicer IMO:

```lua
HarkonnenAttackWaves =
{
	Easy = 3,
	Normal = 6,
	Hard = 9,
}
```

Other examples are in gdi05a.

**Comment by obrakmann** [comment]
Looks fine, thanks. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/fd8c0335adc31600de31ca9dfb3adffd9d47ef38)

**Pull request merged**